### ISSUE TITLE
Fastify-html whitespace issue

### ISSUE THREAD

**Issue by pstachula-dev** [platformatic/ssr-performance-showdown#27]
I found a big problem with whitespace characters, which increase the size of the final server response by 25% (197kb vs 149kb)

With this change, I increased the average number of requests from 968req/s to 1033req/s.

```ts
 return reply.html`<div id="wrapper">
      !${tiles.map(({ x, y }) => (
        server.html`<div class="tile" style="left: ${x.toFixed(2)}px; top: ${y.toFixed(2)}px"></div>`
      ))}
    </div>`
```

**Comment by mcollina** [comment]
cc @gurgunday

**Comment by gurgunday** [comment]
Yeah, seems like it really helped for this benchmark, normally a compressor like brotli or gzip would help as well

Not sure if adding a minifier would make it faster or slower in the end

**Comment by pstachula-dev** [comment]
Minifer should be made in the new compilation step, like all other frameworks, but this introduces a new step, so the issue of DX vs. Perf.

**Comment by gurgunday** [comment]
Oh yeah, I see, and I can actually imagine it in `fastify-html` or `ghtml` one day if we can make it simple enough
